server/helperFunctions.js

```diff
@@ -0,0 +1,48 @@
+const { LoremIpsum } = require('lorem-ipsum');
+const { movies } = require('./movieList.js');
+
+const bgPath = 'https://featurefilm.s3-us-west-2.amazonaws.com/bg/wudu-background-';
+const featurePath = 'https://featurefilm.s3-us-west-2.amazonaws.com/f/wudu-feature-';
+const tagsArray = ['comedy', 'horror', 'drama', 'holiday special', 'action', 'adventure', 'scifi', 'fantasy', 'thriller', 'animated'];
+const mpaaRatings = ['G', 'PG', 'PG-13', 'R', 'NC-17', 'Unrated'];
+const movieNamesArray = movies.split('\n');
+const getRandomNumberTo100 = function () {
```

Comment:
You could combine all of these random number generators into a function that takes the max as a parameter

`getRandomNumber = (max) => { return Math.floor(Math.random() * max) + 1 }`